desktop/src/main/kotlin/io/spine/examples/pingh/desktop/DurationExts.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2024, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.examples.pingh.desktop
+
+import com.google.protobuf.Duration
+import com.google.protobuf.util.Durations
+
+private const val secPerMin = 60
+private const val minPerHour = 60
+
+/**
+ * Returns the number of minutes in this duration.
+ *
+ * @return The number of minutes, from 0 to 59 inclusive.
+ */
+internal fun Duration.minutes(): Int = (Durations.toMinutes(this) % minPerHour).toInt()
```

Comment:
This method is both named and documented in a misleading way. We don't convert the `Duration` to minutes. 

I would rename it into something like
```
Duration.minutesPart()
```
or
```
Duration.minutesOfHour()
```
(the latter is from [Google APIs](https://github.com/googleapis/googleapis/blob/master/google/type/timeofday.proto#L36)).